### ISSUE TITLE
unused parameter ‘dynamic_smem’

### ISSUE THREAD

**Issue by pelesh** [LLNL/RAJA#519]
I am getting warning `unused parameter ‘dynamic_smem’` when building RAJA code with CUDA execution policy. I am not using CUDA shared memory, so I guess this parameter is indeed unused. Is there a way to prevent these warnings? This happens for every RAJA lambda function with CUDA execution policy. I am using RAJA 0.6.0.

**Comment by DavidPoliakoff** [comment]
@trws @ajkunen , do we have a KNOWN_UNUSED flag on our parameters, for [here](https://github.com/LLNL/RAJA/blob/bb3c57449e1e7f07df03abe36f83e72b97730eda/include/RAJA/policy/cuda/MemUtils_CUDA.hpp#L239)? This would solve it

**Comment by DavidPoliakoff** [comment]
Thanks for reporting, @pelesh , we should be able to fix that up

**Comment by rhornung67** [comment]
@DavidPoliakoff yes, in RAJA/include/RAJA/util/macros.hpp, there is RAJA_UNUSED_ARG (similarly for RAJA_UNUSED_ARG.

**Comment by DavidPoliakoff** [comment]
I'll put in the PR, thanks

**Comment by rhornung67** [comment]
👍

**Comment by rhornung67** [comment]
@DavidPoliakoff can this issue be closed? If so, please close it.

**Comment by DavidPoliakoff** [comment]
@rhornung67 , merging my PR automatically closes this bug

**Comment by DavidPoliakoff** [comment]
@rhornung67 : if it's not clear, I don't have merge access on RAJA, so I can't merge this myself. I'll need somebody with merge privileges to do that for me.

**Comment by davidbeckingsale** [comment]
I'll get on it once the CI is done, thanks @DavidPoliakoff, I had not realized that you couldn't merge.

**Comment by rhornung67** [comment]
@DavidPoliakoff I added you to the team. You should have merge privileges now.

**Comment by DavidPoliakoff** [comment]
:+1: , either Beckingsale and I will get it once CI is done
